drivers/dma/Kconfig.andes_atcdmac300

```diff
@@ -0,0 +1,13 @@
+# Andestech ATCDMAC300 configuration options
+# Copyright (c) 2021 Andes Technology Corporation.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+config DMA_ANDES_ATCDMAC300
+	bool "Using Andes ATCDMAC300 DMA driver"
+	default y
+	depends on DT_HAS_ANDESTECH_ATCDMAC300_ENABLED
+	depends on !(ICACHE || DCACHE) || NOCACHE_MEMORY
```

Comment:
can you explain a bit what's going on with the cache? This looks very suspicious.